### ISSUE TITLE
Create a default login credentials for admin test user

### ISSUE THREAD

**Issue by asdfadrian** [RocketChat/Rocket.Chat#5283]
Hey guys Im creating unit tests with selenium. Is there any chance you can post the credentials for Rocket.Cat's info? It would greatly make things much smoother.

Thanks!

**Comment by engelgabriel** [comment]
@asdfadrian use the following environment variables when starting the Rocket.Chat server for the first time, so an admin user will be created for your tests:

```
export ADMIN_USERNAME="admin-test"
export ADMIN_EMAIL="admin-test@rocket.chat"
export ADMIN_PASS="admin-test"
export DISABLE_ANIMATION="true"
```

**Comment by engelgabriel** [comment]
Rocket.Cat does not have a default password.

**Comment by MartinSchoeler** [comment]
@asdfadrian actually you should use only the `DISABLE_ANIMATION="true"` when starting your server. The the env variables for admin should be used when you start the tests with the command `meteor npm run chimp-test` and already have a admin user created, then you can add this admin user info to the vars: 
```
ADMIN_USERNAME="your admin username"
ADMIN_EMAIL="your admin email"
ADMIN_PASS="your admin password"
``` 
If you are running the tests with a clear database you can start the tests with no env vars, because the first user created is a admin

**Comment by MartinSchoeler** [comment]
it would be something like this: `DISABLE_ANIMATION=true meteor` for starting the server
and  `ADMIN_USERNAME="your admin username" ADMIN_EMAIL="your admin email" ADMIN_PASS="your admin password" meteor npm run chimp-test` (with the server running )

**Comment by a5his** [comment]
@MartinSchoeler 

While running the test with and without env variable, both failed. 

<img width="1143" alt="screen shot 2016-12-23 at 11 00 43 pm" src="https://cloud.githubusercontent.com/assets/330362/21459201/4bdc6124-c964-11e6-8049-44ae6d67aa71.png">

Also, running the test without env variable admin user creation is failing. It says email already exists though the test data has only one admin user details in `user.js`.

**Comment by engelgabriel** [comment]
We are fixing this :)

**Comment by engelgabriel** [comment]
@asdfadrian  and @a5his , @rodrigok will create a default admin user when RC is running with a TESTMODE=true environment variable.

**Comment by rodrigok** [comment]
Closed via #5329
